client/app/models/pas-form.js

```diff
@@ -1,9 +1,14 @@
-import Model, { belongsTo, attr } from '@ember-data/model';
+import Model, { belongsTo, hasMany, attr } from '@ember-data/model';
 
 export default class PasFormModel extends Model {
-  @belongsTo
+  @belongsTo({ async: false })
```

Comment:
Wanted to learn what the tradeoff / benefits of making our model relationships synchronous.  Found [this article](https://embermap.com/notes/83-the-case-against-async-relationships), and it seems like a good way to make things easier to reason about.

Reply:
Yes! That's exactly why. I believe @andycochran also had trouble because we had one of these rels set as asynchronous, it was returning a Proxy object.

Frankly, we should go through all of our apps and set it to false. And probably add a lint rule.